### PR TITLE
GraphQL: Mutation for adding wishlist items to the cart

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) adds documentation for the addWishlistItemsToCart mutation.

Related PRs: 
https://github.com/magento/magento2/pull/31584
https://github.com/magento/partners-magento2ee/pull/449

Issue:
https://github.com/magento/magento2/issues/31331

## Affected DevDocs pages

-  ...

## Links to Magento source code

-  ...

### CODE REVIEW

**Review by dobooth — APPROVED** [review]

File: src/guides/v2.4/graphql/mutations/add-wishlist-items-to-cart.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ Attribute |  Data Type | Description
 
 Type | Description
 --- | ---
-`INSUFFICIENT_STOCK` | The requested product is out of stock
+`INSUFFICIENT_STOCK` | The requested quanity of a product is greater than the quantity available
```
Comment (BarnyShergold) [inline src/guides/v2.4/graphql/mutations/add-wishlist-items-to-cart.md:152]:
```suggestion
`INSUFFICIENT_STOCK` | The requested quantity of a product is greater than the quantity available
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

**Comment by BarnyShergold** [comment]
@keharper - There's a small spelling mistake to correct on this PR

File: src/guides/v2.4/graphql/mutations/add-wishlist-items-to-cart.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+## Output attributes
+
+The `addWishlistItemsToCart` mutation returns the status of the operation as well an array of any errors that occurred.
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`add_wishlist_items_to_cart_user_errors` | [[WishlistCartUserInputError!](#WishlistCartUserInputError)] | Indicates why the attempt to add items to the wish list was not successful
+`status` | Boolean! | Indicates whether the attempt to add items to the cart was successful
+`wishlist` | [Wishlist!](#Wishlist) | Contains the wish list with all items that were successfully added
```
Comment (dthampy) [inline src/guides/v2.4/graphql/mutations/add-wishlist-items-to-cart.md:133]: I wonder if the description should be something like " Contains items in the wishlist after adding to cart" . But it is up to you @keharper if we need to align the description with other mutations where "wishlist" is being returned in the response or if we need to update description depending on the context

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
running tests

**Review by dthampy — APPROVED** [review]

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
running tests

**Pull request merged**